packages/page-accounts/src/Sidebar/AccountMenuButtons.tsx

```diff
@@ -85,12 +87,14 @@ function AccountMenuButtons ({ className = '', flags, isEditing, isEditingName,
         )
         : (
           <Button.Group>
-            <Button
-              icon='paper-plane'
-              isDisabled={isEditing}
-              label={t<string>('Send')}
-              onClick={toggleIsTransferOpen}
-            />
+            {isFunction(api.api.tx.balances?.transfer) && (
```

Comment:
This really should be enough, I'm guessing it isn't so will take a look at why not after this is in. (e.g. the re-checks for the functions inside the `Transfer` file shouldn't be needed, it really should not try to execute anything on it)

Reply:
Ah yeah good point, I made those changes first but you're probably right that this would be enough.